### DISCUSSION TITLE
New Catch Fun Mod Idea: Wrap

### DISCUSSION THREAD

**Discussion by TomyDoesThings** [ppy/osu#25036] [Ideas]
This mod would allow the catcher to wrap around the screen, making for a neat twist to how a Catch map would be played.

There is an invisible wall on the left and right side of the Catch playfield that normally prevents the catcher from continuing to move in that direction. This "Wrap" mod would make those invisible walls visible and look teleporter-like. As the catcher enters one of them, they will start sticking out in the other one.

This would also mean that hyper fruits that normally were needed to hit a pattern could turn into regular fruits, as the catcher would theoretically at maximum only need to move through half the screen rather than occasionally moving a full distance. So warning: this can affect SR as well.

Only Catch would have this as this mod would be incompatible for standard players that use tablet, and taiko along with mania do not have a cursor or catcher that can wrap around the screen. I love Catch as it is already, and this mod would add so much more playability to its maps.

**Comment by peppy** [comment]
Should be pretty fun, and implementing shouldn't be too hard.
